src/pivot-table/components/helpers/get-item-size-handler.ts

```diff
@@ -31,24 +29,22 @@ export const getRowHeightHandler =
   };
 
 export const getColumnWidthHandler =
-  ({
-    list,
-    isLastRow,
-    layoutService,
-    allMeasuresWidth,
-    getMeasureInfoWidth,
-  }: ColumnWidthHandlerProps): ItemSizeHandler =>
+  ({ list, isLastRow, getRightGridColumnWidth }: ColumnWidthHandlerProps): ItemSizeHandler =>
   (colIndex: number) => {
     const cell = isLastRow ? list[colIndex] : Object.values(list)[colIndex];
-    const measureInfoCount = layoutService.layout.qHyperCube.qMeasureInfo.length;
+    // const measureInfoCount = layoutService.layout.qHyperCube.qMeasureInfo.length;
 
-    if (colIndex === 0 && cell?.x > 0) {
-      return ((cell.leafCount + cell.x) / measureInfoCount) * allMeasuresWidth;
-    }
+    // TODO: This a bit of a special case but if you are on a different page then the first.
+    // Scroll down a bit and expand a node. The "first row" does not exist yet as data has only been
+    // fetched for the rows that are visible to the user.
+    // if (colIndex === 0 && cell?.x > 0) {
+    //   return ((cell.leafCount + cell.x) / measureInfoCount) * allMeasuresWidth;
```

Comment:
Changing this to `return (cell.leafCount + cell.x) * getRightGridColumnWidth();` seem to fix the issue where it fails to render cells at least

Reply:
great, I will test a bit with that